### DISCUSSION TITLE
Add save state and load state function

### DISCUSSION THREAD

**Discussion by usergithub64** [86Box/86Box#2917] [Ideas]
Please add a save state and load state function to 86Box for games, similar to how it was done for DOSBox-X:
https://github.com/joncampbell123/dosbox-x/issues/842

**Comment by OBattler** [comment]
In 86Box, that's going to be rather more complicated due to all the devices the state would have to be saved for.

**Comment by techflashYT** [comment]
I second this, it would be extremely useful, especially if it can be automated somehow.  I am trying to do a project with an emulated 486 that will take over 100 days nonstop.  The chances that the host machine will experience some problem during that time is immense, so having routine automated backups of the running guest would be critical.
Can we get an update on if there's any chance this is happening, since it's been so long since the initial posting? @OBattler
